src/popup/pages/Send/index.tsx

```diff
@@ -121,30 +129,35 @@ const TokenField = observer(({ classes, store: { sendStore } }: any) => (
 ));
 
 const AmountField = observer(({ classes, store: { sendStore }, onEnterPress }: any) => (
-  <div className={classes.amountContainer}>
-    <div className={classes.amountHeadingContainer}>
-      <div className={classes.amountHeadingTextContainer}>
+  <div className={classes.fieldContainer}>
+    <div className={classes.buttonFieldHeadingContainer}>
+      <div className={classes.buttonFieldHeadingTextContainer}>
         <Heading name="Amount" />
       </div>
-      <Typography className={classes.maxAmountText}>{sendStore.maxAmount}</Typography>
+      <Typography className={classes.fieldButtonText}>{sendStore.maxAmount}</Typography>
       <Button
         color="primary"
-        className={classes.maxButton}
+        className={classes.fieldButton}
         onClick={() => sendStore.amount = sendStore.maxAmount}
       >
         Max
       </Button>
     </div>
     <div className={classes.fieldContentContainer}>
       <TextField
+        className={classes.selectOrTextField}
         fullWidth
         type="number"
         multiline={false}
         placeholder={'0.00'}
         value={sendStore.amount}
         InputProps={{
+          classes: {
+            input: classes.fieldInput,
+          },
+          className: classes.fieldTextOrInput,
```

Comment:
are you not able to change the fontSize by putting it in the `fieldInput`? if able to, lets just use one.

Reply:
we can change the fontSize that way, and it would remove line 158,  but all of the other input fields on the page are using className: classes.fieldTextOrInput to set their fontsize.
I think it's better to keep a consistent. What do you think?